be1-go/hub/organizer.go

```diff
@@ -205,6 +274,33 @@ func (o *organizerHub) Start(done chan struct{}) {
 	}
 }
 
+func (o *organizerHub) verifyJson(byteMessage []byte, schemaName string) error {
```

Comment:
Thinking about it again, you might want to refactor this part because it's going to be common across both Organizer and Witness Hubs

Reply:
Could I let it like this in this PR and change it in #315 where I would replace `o *organizerHub` with `b *baseHub` once this PR is merged?

Reply:
Might even be worth having a separate package that deals with validation exclusively. But yeah, that's something that can be taken up in another PR 👍